detect/synthetic-monitoring/playwright-checks/environment-variables.mdx

```diff
@@ -5,28 +5,121 @@ description: "Customize your Playwright Check Suite runs based on the execution
-- **CI**: Set to `1` for the following check runs:
-  - CLI runs via `npx checkly test` or `npx checkly trigger`.
-  - Check runs that are triggered by [deployments](/integrations/ci-cd/github/deployments).
-
-The following variables are mostly for informational and debugging purposes and shouldn't be used to influence test behaviour.
-
-- `ACCOUNT_ID`: The UUID of the Checkly account as found in the URL.
-- `CHECK_NAME`: The name of the check.
-- `CHECKLY_CHECK_ID`: The UUID of the check as found in the URL.
-- `CHECKLY_REGION`: The region in which the check was executed.
+<Note>
+  For information on creating and managing environment variables and secrets (including the difference between variables and secrets, inheritance, and where to set them), see the [Environment Variables documentation](/platform/variables).
+</Note>
+
+## Built-in variables available in Playwright Check Suites
+
+Checkly sets the following environment variables on every Playwright Check Suite run:
+
+| Variable | Description |
+|----------|-------------|
+| `CHECKLY` | Set to `1` for all check runs executed by Checkly.<br/>Useful to distinguish Checkly runs from local runs. |
+| `CHECKLY_RUN_SOURCE` | Indicates how the check was triggered.<br/>Useful to customize the nature of your test depending on how it is run.<br/>See [CHECKLY_RUN_SOURCE values](#checkly-run-source-values) below. |
+| `CI` | Set to `1` for CLI runs (`npx checkly test` or `npx checkly trigger`) and [deployment](/integrations/ci-cd/github/deployments)-triggered checks. |
+| `ACCOUNT_ID` | The UUID of the Checkly account. |
+| `CHECK_NAME` | The name of the check. |
+| `CHECKLY_CHECK_ID` | The UUID of the check. |
+| `CHECKLY_REGION` | The region where the check was executed. |
+
+### `CHECKLY_RUN_SOURCE` values
+
+| Value | Description |
+|-------|-------------|
+| `CLI_DEPLOY` | Used for the first run of Checks deployed using `npx checkly deploy`. |
+| `DEPLOYMENT` | Used for Check runs triggered as part of a [CI/CD deployment](/integrations/ci-cd/github/deployments). |
+| `GROUP_RUN_ALL` | Used for Check runs triggered from a Group's edit screen, by clicking the "Run all checks" button. |
+| `SCHEDULE_NOW` | Used for Check runs triggered manually by clicking "Schedule now" in the webapp. |
+| `SCHEDULER` | Used for Check runs triggered as part of their regular schedule. |
+| `TEST_NO_RECORD` | Used for Check runs triggered by the `npx checkly test` CLI command. |
+| `TEST_RECORD` | Used for Check runs triggered by the `npx checkly test --record` or `npx checkly pw-test` CLI commands. |
+| `TRIGGER_API` | Used for Check runs triggered by the API. |
+| `TRIGGER_NO_RECORD` | Used for Check runs triggered by the `npx checkly trigger` CLI command. |
+| `TRIGGER_RECORD` | Used for Check runs triggered by the `npx checkly trigger --record` CLI command. |
+
+## Use built-in variables
+
+Common ways to use Checkly's built-in environment variables:
+
+### Skip tests in production monitoring
```

Comment:
I'm not sure if we should promote the usage of `process.env.CHECKLY` to skip tests. It's tackling the topic of test selection from the other side. And while it works I'd say we should promote opting in via projects or tags and a proper PCS definition. :)